drltrace/drltrace.config

```diff
@@ -0,0 +1,8 @@
+; The syntax of this config file is simple. To add new function, it's enough to specify
+; each argument of the function, function name itself and return type separated by commas.
```

Comment:
Hmm, if we know this syntax won't support some types, it seems best to consider now whether some other syntax would be better.

There should be a version number to allow changing or extending the syntax, if the idea is to start with something we know is going to need expansion (even if not, versioning is always a good idea).

It also seems best to mention that some types are not yet supported.